GameData/RP-1/Contracts/Commercial Applications 1/EarlyComSat-CA.cfg

```diff
@@ -144,7 +144,7 @@ CONTRACT_TYPE
 				name = Duration
 				type = Duration
 
-				duration = 2m
+				duration = 30d
```

Comment:
Since commercial sats are destroyed on contract complete there's no need for more duration than 2m. Sorry I wasn't clear, I meant to pump up the duration on the EOS side, on the CA side 2m stability checks are fine.

Reply:
Easy change.